test/stdlib/UnicodeMetadata.swift

```diff
@@ -1,4 +1,4 @@
-// RUN: %target-build-swift -module-name="日本語01" %s -o %t.out
+// RUN: %target-build-swift "@%/S/Inputs/Unicode.resp" %s -o %t.out
```

Comment:
The check `// CHECK: [日本語01.myClass]` in this file is checking against UTF-8 bytes. This prints out UTF-8 bytes on Windows, so that passes the check, but I wasn't entirely sure if the intended behaviour. Actually running it prints out a badly encoded mess.
```
S:\b\swift> &"s:\b\swift\test-windows-x86_64\stdlib\Output\UnicodeMetadata.swift.tmp.out.exe"
[µùÑµ£¼Φ¬₧01.myClass] 
```

Reply:
Interesting, but probably correct. I think we assume our output streams are UTF-8ish now. @milseman?

Reply:
I haven't made any changes there. I believe it's been the case since before I started working on the project.

Reply:
Yeah, just wanted to know if that was an assumption we were already making. Thanks!